Backend/ServiceLayer/Services/ResetService.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using DataAccessLayer.Repositories;
+using MimeKit;
+using MailKit;
+using MailKit.Net.Smtp;
+
+
+namespace ServiceLayer.Services
+{
+    public class ResetService
+    {
+
+        #region Constants for the reset service
+        //Variable for how long the token is supposed to be live
+        private const double TimeToExpire = 5;
+        //Constant for the password reset email subject line
+        private const string resetPasswordSubjectString = "KFC SSO Password Reset";
+        //URL to append the reset ID to. Sent to the user in the email body.
+        private const string resetControllerURL = "kfcsso.com/api/reset/resetpassword/?id=";
+
+        #endregion
+
+        private ResetRepository _resetRepo;
+        public ResetService()
+        {
+            _resetRepo = new ResetRepository(new DatabaseContext());
+        }
+
+        //Function to create token in the system
+        public string CreateResetID()
```

Comment:
Need to change this to create a new GUID

Reply:
Nah, this is where he's creating his "token" - See my comments above. GUID (ID) for the object and the "token" need to be two separate things. The ID is used to identify the object within the database, while the token is a cryptographically secure string.

Reply:
Please take a look at our session service. This type of logic has already been created. If it's not already reusable from there, make it reusable so we only have token generation logic in one place. As of now, this is a duplicate.